Fail to build with LLVM8/Clang8.

## Build Process

FreeBSD [Port](https://svnweb.freebsd.org/ports/head/x11/polybar/).

[Complete build log](http://package18.nyi.freebsd.org/data/headamd64PR236062-default/2019-03-01_06h42m53s/logs/errors/polybar-3.3.0.log).

```
--- Description ---
$ poudriere jail -cj clang8 -v projects/clang800-import -m svn+https
$ poudriere testport -j clang8 x11/polybar
[...]
In file included from src/components/bar.cpp:22:
In file included from include/x11/tray_manager.hpp:15:
include/x11/tray_client.hpp:18:16: error: explicitly defaulted copy assignment
operator is implicitly deleted [-Werror,-Wdefaulted-function-deleted]
  tray_client& operator=(tray_client& c) = default;
               ^
include/x11/tray_client.hpp:38:15: note: copy assignment operator of
'tray_client' is implicitly deleted because field 'm_connection' is of
reference type 'polybar::v3_3_0::connection &'
  connection& m_connection;
              ^
1 error generated.
```

IDK C++ but I've asked a friend:

> To fix it would either need to write the assignment operator (explicitly *also* copying the reference), or change the type of connection from reference to something that is implicitly copyable (e.g. pointer, some kind of reference wrapper, ...)

Thank you.

--- a/include/x11/tray_client.hpp
+++ b/include/x11/tray_client.hpp
@@ -14,8 +14,8 @@ struct xembed_data;
 class tray_client {
  public:
   explicit tray_client(connection& conn, xcb_window_t win, unsigned int w, unsigned int h);
-  tray_client(const tray_client& c) = default;
-  tray_client& operator=(tray_client& c) = default;
+  tray_client(const tray_client& c) = delete;
+  tray_client& operator=(tray_client& c) = delete;
 
   ~tray_client();
 

